types/block_bench_test.go

```diff
@@ -6,6 +6,14 @@ import (
 	"github.com/NebulousLabs/Sia/encoding"
 )
 
+func BenchmarkEncodeBlock(b *testing.B) {
```

Comment:
can you include a comment with historical data? Also, can it be named `BenchmarkEncodeEmptyBlock`?  Same with the next function, the docstring has the right name but the actual function name is wrong.